src/index.html

```diff
@@ -1,13 +1,60 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com">
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" 
+      rel="stylesheet">
+  </head>
+  <body>
+    <div class="container" data-qa="card">
+      <img 
+        class="container__image"
+        src="./images/imac.jpeg" 
+        alt="Imac"
+      >
+
+      <h2>APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      
+      <div class="container__product-code">
+        Product code: 195434
+      </div>
+
+      <div class="container__feedback">
+        <div class="stars">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+        <span class="container__reviews">
+          Reviews: 5
+        </span>
+      </div>
+
+      <div class="container__price">
+        <span class="container__price-label">Price:</span>
+        <span class="container__price-value">$2,199</span>
+      </div>
+
+      <a
+        class="container__button-buy"
+        data-qa="hover"
+        href="#"
+      >
+      BUY
```

Comment:
```suggestion
      Buy
```
It's better to use `text-transform` in CSS